pages/_app.js

```diff
@@ -1,10 +1,11 @@
 import GlobalStyle from "../styles";
+import projects from "@/lib/projects.js";
```

Comment:
since you are using "@" here, you could consider to do it in any cases (instead of "..")